contracts/contracts/Sapphire.sol

```diff
@@ -29,6 +29,8 @@ library Sapphire {
         0x0100000000000000000000000000000000000101;
     address internal constant SHA512 =
         0x0100000000000000000000000000000000000102;
+    address internal constant SHA384 =
```

Comment:
1. While you're at it, can you document this with `@dev` tag and point to the sapphire-paratime rust code where this is implemented? And the same for other constants ;)
2. This adds another 20-byte value to the contract code. Any considerations about the Sapphire contract size? Not sure, if solidity supports this, but could we use a single instance of the contract deployed by ourselves and then have others just linking to it?

Reply:
Re: 2. as it's a `constant` it won't increase contract size. When `Sapphire.sol` is included the optimizer will only emit bytecode for the functions which are called.